app/adapters/file-provider.ts

```diff
@@ -0,0 +1,14 @@
+import OsfAdapter from './osf-adapter';
+
+export default class FileProvider extends OsfAdapter {
+    pathForType = function(): string {
```

Comment:
This should be class method syntax like the original, but with the type annotation added.